Content.Server/GameObjects/Components/Weapon/Melee/StunbatonComponent.cs

```diff
@@ -33,9 +47,30 @@ public class StunbatonComponent : MeleeWeaponComponent, IUse
         [ViewVariables(VVAccess.ReadWrite)]
         private float _slowdownTime = 5f;
 
+        [ViewVariables(VVAccess.ReadWrite)] public float Wattage { get; set; } = 1000;
```

Comment:
I'm not sure saying "wattage" makes much sense here considering power cells don't consume power over time. Maybe `EnergyPerUse`?

Reply:
Can do